openpype/hosts/houdini/api/lib.py

```diff
@@ -558,19 +558,36 @@ def get_frame_data(node):
         dict: frame data for star, end and steps.
 
     """
+    if asset_data is None:
+        asset_data = {}
+
+    if log is None:
+        log = self.log
+
     data = {}
 
     if node.parm("trange") is None:
-
+        log.debug(
+            "Node has no 'trange' parameter: {}".format(node.path())
+        )
         return data
 
     if node.evalParm("trange") == 0:
-        self.log.debug("trange is 0")
+        log.debug(
+            "Node '{}' has 'Render current frame' set. "
+            "Time range data ignored.".format(node.path())
+        )
         return data
 
-    data["frameStart"] = node.evalParm("f1")
-    data["frameEnd"] = node.evalParm("f2")
-    data["steps"] = node.evalParm("f3")
+    data["frameStartHandle"] = node.evalParm("f1")
+    data["handleStart"] = asset_data.get("handleStart", 0)
```

Comment:
is `handleStart` on the asset returned as an int?

Reply:
yes, it's an int.